README.md

```diff
@@ -136,6 +138,10 @@ If you're confident it's a new bug and have confirmed that someone else is facin
 
 If you think you have found a vulnerability, please report responsibly. Don't create GitHub issues for security issues. Instead, please send an email to `security@freecodecamp.org` and we'll look into it immediately.
 
+### Setting Up Project Locally
```

Comment:
Maybe move this down the page?  Surely most people are going to want to learn how to contribute first, so that should be higher.

Reply:
Sure. Brought this up as I was looking for information on how to set up the repo locally. The README is the first place I'd go to

Reply:
Understood.  Since we're going to deal with linting properly soon (hopefully!) could you take those changes out?  If you could just leave the updated copyright and the new link, that would be great.  Then I can ping someone on the core team to give it a final review.